### PR TITLE
DSNP on Frequency Publishing

### LINKED ISSUE #188: [DSNP] Frequency Publishing
**Frequency Creation of New Announcements**

As a publisher of announcements, I want to know how to publish different announcement types

**Frequency Real Time Consumption**

As a user, I want to know when a new announcement occurs in real time.

**Batch File Retrieval**

As a user, I need to know how to retrieve a batch file.

**Ordering**

As a user, I need to know how to order all messages.

**Historical Data Retrieval**

As a user, I want to know how to get announcement I have missed.

**Notes**
- https://github.com/LibertyDSNP/schemas has all the schemas

### PR DESCRIPTION
**Pull request by wilwade**
Problem
=======
Initial build out of the Announcement publishing portion of the Frequency Implementation.

Link to GitHub Issue(s): Closes #188 

Change summary:
---------------
- [x] Wrote initial draft of /Frequency/Publishing.md
- [x] Spelling words
- [x] Added comment to update links from rust documentation to docs.frequency.xyz when that is done.

Screenshots (optional):
-----------------------
image

### CODE REVIEW

File: pages/Frequency/Publishing.md [hunk 1]
```diff
@@ -1,24 +1,56 @@
...
 
-| Scheme | Description | Reference | DSNP Version Added |
-| ------ |------------ | --------- | ------------------ |
-| HTTPS | Hypertext Transfer Protocol Secure | [RFC2818](https://datatracker.ietf.org/doc/html/rfc2818) | 1.0 |
+## DSNP Batched Announcements
+
+Frequency uses [DSNP Batch Publications](../DSNP/BatchPublications.md) for Announcements that are batched.
+The parquet file is stored on IPFS, but it is discovered through the Frequency Message.
+
+Frequency Messages maintain the data of when the data was published and what Providers MSA published it.
```
Comment (shannonwells) [inline pages/Frequency/Publishing.md:30]: > Frequency Messages maintain the data of when the data was published...

Not sure what you mean here.

File: pages/Frequency/Publishing.md [hunk 2]
```diff
@@ -1,24 +1,56 @@
...
+It is the publisher's responsibility to maintain the IPFS pin so that the batch file is continuously available.
```
Comment (shannonwells) [inline pages/Frequency/Publishing.md:31]: Will the intended audience know what an IPFS pin is?  Maybe this should link to an IPFS reference here or at the bottom.

File: pages/Frequency/Publishing.md [hunk 3]
```diff
@@ -1,24 +1,56 @@
...
-...
+Frequency Messages are well ordered within a Schema
+
+1. Frequency: Block number ascending
+2. Frequency: Transaction index ascending
+3. DSNP Standard: Order Announcements in a Batch Publication File by row appearance order
+
+### Ordering Across Schemas
+
+Frequency does not provide complete order information for Messages across Schemas.
```
Comment (shannonwells) [inline pages/Frequency/Publishing.md:45]:
```suggestion
Frequency does not provide complete ordering metadata for Messages across Schemas.
```

Comment (shannonwells) [reply pages/Frequency/Publishing.md:45]: Maybe "ordering" or "ordering information"....

**Review by shannonwells — APPROVED** [review]
Some nitpicks, nothing blocking.

File: pages/Frequency/Publishing.md (hunk 1 above)
Comment (wilwade) [reply pages/Frequency/Publishing.md:30]: @shannonwells How does this work?
```suggestion
Frequency Messages maintain the metadata of where and when the Batch Publication was published and which Provider MSA published it.
```

File: pages/Frequency/Publishing.md (hunk 3 above)
Comment (aramikm) [inline pages/Frequency/Publishing.md:45]: We do provide order across different schemas in the same block. `Index` stored on each Message is a total index in that block across all schemas.

**Review by aramikm — APPROVED** [review]
I added one comment

**Pull request merged**